Adjustments for role_binding in SG for CI

leifmadsen opened the pull request:
Update the stf-run-ci playbooks to account for the namespace:
placeholder that has been added to role_binding.yaml as part of the
oauth-proxy work. Before this change, the deployment of SGO would fail
in the CSV due to a missing ClusterRoleBinding attaching to the
service-account in the appropriate namespace. This change causes the
role_binding.yaml to be updated to the installation namespace so that
the service account is referenced in the role binding properly.

Also sneak in a small change that reverts the local in place change to
the role_binding.yaml for the local repository.

leifmadsen commented:
Currently testing this again with these changes merged into local branch holding #326 

```bash
OCP_PROJECT='service-telemetry' EPHEMERAL_STORAGE=true ./deploy/quickstart.sh
```

csibbitt commented on build/stf-run-ci/tasks/setup_stf_local_build.yml:
```diff
@@ -36,6 +42,9 @@
 - name: Load Smart Gateway Operator CSV
   shell: oc apply -f working/smart-gateway-operator-bundle/manifests/smart-gateway-operator.clusterserviceversion.yaml -n "{{ namespace }}"
 
+- name: Revert local change to role_binding.yaml
```
Oh thanks, this will save me some minutes in the future I'm sure :)

leifmadsen replied:
Bah! I put it in the wrong spot :) This needs to be done after loading the STO RBAC...

leifmadsen commented:
```bash
TASK [stf-run-ci : Generate Service Telemetry Operator CSV] ****************************************************************************************************************
changed: [localhost]

TASK [stf-run-ci : Replace namespace in STO role binding] ******************************************************************************************************************
changed: [localhost]

TASK [stf-run-ci : Load Service Telemetry Operator RBAC] *******************************************************************************************************************
changed: [localhost] => (item=service_account.yaml)
changed: [localhost] => (item=role.yaml)
changed: [localhost] => (item=role_binding.yaml)
changed: [localhost] => (item=olm-catalog/service-telemetry-operator/manifests/infra.watch_servicetelemetrys_crd.yaml)

TASK [stf-run-ci : Revert local change to role_binding.yaml] ***************************************************************************************************************
changed: [localhost]
```

leifmadsen commented:
Re-tested and this is working as intended now.
